geode-management/src/main/java/org/apache/geode/management/configuration/MemberConfig.java

```diff
@@ -154,6 +154,10 @@ public String getId() {
     return groups;
   }
 
+  public void setGroups(List<String> groups) {
```

Comment:
the rest of the config objects don't typically have a setter for any list objects. They usually do getList().add to modify the list. But this is minor. Just want to point out the consistency.

Reply:
I think the pattern of getting a thing, `getList` and then modifying it is not very intuitive. Is there a middle ground we can do instead?

Reply:
when doing a setList(List myList), you have access to the myList object, and any changes to myList might unintentionally change the list content of your config object. to get it and then modify it make it a conscious thing to change it.  Let's keep it consistent with the rest of the config objects and use getList().clear and getList().addAll() to change the content of the config list.

Reply:
Probably the best thing to do is follow the existing convention for now.  If we want to change it, we should make that change everywhere at once and as a separate PR.  But to the points above:

getList().add() is not really any safer, any code can still get the list and change it out from under the config object at any time.  to prevent that we would need to make getList() return an immutable list, and of course change set(List) to make a copy.